File: setup_splash.py
import base64
import os

# Create assets directory
os.makedirs('assets', exist_ok=True)

# Minimal 1x1 dark blue PNG encoded in base64
# This is a valid PNG file with color #1A1A2E
png_b64 = 'iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAADUlEQVR42mP8z8DwHwAFBQIAX8jx0gAAAABJRU5ErkJggg=='

with open('assets/splash_logo.png', 'wb') as f:
    f.write(base64.b64decode(png_b64))

print('Created assets/splash_logo.png')

File: setup_splash.dart
import 'dart:io';
import 'dart:convert';

void main() {
  // Create assets directory
  final assetsDir = Directory('assets');
  if (!assetsDir.existsSync()) {
    assetsDir.createSync(recursive: true);
  }

  // Minimal 1x1 dark blue PNG (base64 of valid PNG with color #1A1A2E)
  const base64Png = 'iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAADUlEQVR42mNkYPhfDwAChwGA60e6kgAAAABJRU5ErkJggg==';

  final bytes = base64Decode(base64Png);
  File('assets/splash_logo.png').writeAsBytesSync(bytes);
  print('Created assets/splash_logo.png');
}
